lib/devicetrust/testenv/testenv.go

```diff
@@ -114,3 +114,13 @@ func New() (*E, error) {
 	ok = true
 	return e, nil
 }
+
+type FakeDevice interface {
```

Comment:
Add a comment, mention that this interfaces over the native methods?

Reply:
d81036d196